packaging/version.py

```diff
@@ -282,16 +310,50 @@ def base_version(self):
         return "".join(parts)
 
     @property
+    def epoch(self):
+        return self._version.epoch
+
+    @property
+    def release(self):
+        return self._version.release
+
+    @property
     def local(self):
         version_string = str(self)
         if "+" in version_string:
             return version_string.split("+", 1)[1]
 
     @property
+    def local_info(self):
+        return self._version.local
+
+    @property
+    def prerelease(self):
+        return self._version.pre
+
+    @property
     def is_prerelease(self):
         return bool(self._version.dev or self._version.pre)
 
     @property
+    def development(self):
+        if self._version.dev is not None:
+            return self._version.dev[1]
+        else:
+            return None
+
+    @property
+    def is_development(self):
+        return bool(self._version.dev)
+
+    @property
+    def postrelease(self):
+        if self._version.post is not None:
+            return self._version.post[1]
```

Comment:
`1.2.post` is normalized to `1.2.post0` when parsing, so `Version('1.2.post').postrelease` is `0`.